client/my-sites/pages/virtual-page/index.tsx

```diff
@@ -168,6 +174,12 @@ const VirtualPage = ( {
 	);
 };
 
+const mapStateToProps = ( state: any, ownProps: Props ) => {
+	return {
+		isAdmin: canCurrentUser( state, ownProps.site.ID, 'manage_options' ),
```

Comment:
Should it be `edit_theme_options` as the site editor also checks this one?

Reply:
you can check the template API or `site-editor.php` on wpcom, both of them check the permission via `current_user_can( 'edit_theme_options' )`. Thus, I think we need to align with them

Reply:
Got it